mail_box2.py

```diff
@@ -0,0 +1,108 @@
+class Message:
+    """Message Department, displays all the necessary information within a message.
+    : ivar int id: Message ID.
+    : ivar string body: Message body.
+    : ivar string sender: Name of sender of the message.
+    : ivar string status: Message Read Status - Optional.
+
+    : param int message_id: Identifies the message being produced.
+    : param string message_body: The body of the message produced.
+   : param string message_sender: Name of the current sender of the message.
+   : param string message_status: Status of reading the current message.
+    """
+
+    def __init__(self, message_id, message_body, message_sender, message_status="not read"):
+        self.id = message_id
+        self.body = message_body
+        self.sender = message_sender
+        self.status = message_status
+
+    def __str__(self):
+        """Prints the message.
+        """
+        print("Message from ",  self.sender, ":")
+        print(self.body)
```

Comment:
These should be returned, not printed.
Please read about the __str__ method.
https://www.educative.io/edpresso/what-is-the-str-method-in-python